packages/tasks/src/local-apps.ts

```diff
@@ -82,6 +117,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: isGgufModel,
 		snippet: snippetLlamacpp,
 	},
+	"vllm": {
+		prettyLabel: "vLLM",
+		docsUrl: "https://docs.vllm.ai",
+		mainTask: "text-generation",
+		displayOnModelPage: isGptqModel && isAwqModel,
```

Comment:
how would you define those methods?

Reply:
> how would you define those methods?

In fact, the suggested vLLM method deploys the non-quantized version from the Hugginface repository. All examples of type "text-generation" in the code are GGUF. Any suggestion?

Reply:
Thank you for the PR! Concretely we support a set of `architectures` which is readable from the model data https://github.com/huggingface/huggingface.js/blob/04a0eb4a01f22f61e9a0b87897c448b914ab1833/packages/tasks/src/model-data.ts#L40

https://github.com/vllm-project/vllm/blob/757b62c49560baa6f294310a53032348a0d95939/vllm/model_executor/models/__init__.py#L13-L63

And for quantization method we can read in `config.quantization_config.quant_method` which we support awq, gptq, aqlm, and marlin

https://huggingface.co/TheBloke/zephyr-7B-alpha-AWQ/blob/main/config.json#L28

Reply:
awesome @simon-mo, super clear!

Reply:
i've pushed `2123430` on this PR to type `config.quantization_config.quant_method` which we will parse & pass from the Hub

Reply:
> i've pushed [2123430](https://github.com/huggingface/huggingface.js/commit/2123430a737a92938d41d22b5fc9229b384a12bc) on this PR to type `config.quantization_config.quant_method` which we now parse & pass from the Hub

I made some changes, I need your help to review